lib/rex/post/meterpreter/extensions/stdapi/fs/dir.rb

```diff
@@ -52,9 +52,10 @@ def each(&block)
   #
   # Enumerates all of the files/folders in a given directory.
   #
-  def Dir.entries(name = getwd)
+  def Dir.entries(name = getwd, glob = nil)
     request = Packet.create_request('stdapi_fs_ls')
     files   = []
+    name = name + ::File::SEPARATOR + glob if glob
```

Comment:
::File::Separator will be ```/```. I guess when targeting windows machines should be ```\```. I guess ```/``` is mostly to work. Not sure if there could be cases where it could become a problem. If isn't the case we can forget about it :)